src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String paragraph = System.lineSeparator();
+        StringBuilder builder = new StringBuilder("Report for period ")
+                .append(dateFrom).append(" - ")
+                .append(dateTo);
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d.MM.yyyy");
```

Comment:
It is [recommended](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#if-you-create-a-formatter-make-it-a-constant-field) to make your formatter a constant field